composer.json

```diff
@@ -8,9 +8,15 @@
     "license": "BSD-3-Clause",
     "name": "cloudflare/wordpress-plugin",
     "type": "wordpress-plugin",
+    "repositories": [
```

Comment:
swapped backend to a GitHub source as i don't have access to Packagist until @manatarms returns next week.